### PR TITLE
Updating composer dependencies

### PR DESCRIPTION
**Pull request by mamazu**
The `lakion/api-test-case` package is now replaced with it's successor.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
         "akeneo/phpspec-skip-example-extension": "^4.0",
         "doctrine/orm": "^2.5",
         "gedmo/doctrine-extensions": "^2.4",
-        "lakion/api-test-case": "^3.1.0",
+        "lchrusciel/api-test-case": "^3.1.0",
```
Comment (lchrusciel) [inline composer.json]: Can we update it to '^4.1' as well?

Comment (mamazu) [reply composer.json]: Sure, currently this fails due to some unknown error.

**Comment by mamazu** [comment]
I think the build job failed for some reason. Could you restart it?

**Comment by lchrusciel** [comment]
done

**Comment by lchrusciel** [comment]
Ah, this container was removed from dependencies and added as a suggestion. That's why it's failing.

Ref. https://github.com/lchrusciel/ApiTestCase/blob/master/UPGRADE.md#upgrading-from-v3-to-v4

File: src/Bundle/spec/Controller/RequestConfigurationFactorySpec.php [hunk 2]
```diff
@@ -43,7 +43,7 @@ function it_creates_configuration_from_resource_metadata_and_request(
         $request->headers = $headersBag;
         $request->attributes = $attributesBag;
 
-        $headersBag->get('Accept', null, false)->willReturn([]);
+        $headersBag->get('Accept', null)->illReturn([]);
```
Comment (pamil) [inline src/Bundle/spec/Controller/RequestConfigurationFactorySpec.php]:
```suggestion
        $headersBag->get('Accept', null)->willReturn([]);
```

**Comment by mamazu** [comment]
@pamil This is the new way of doing things. However to preserve backwards compatibilty the `all` function that psalm complains about already has documentation like that.

**Comment by pamil** [comment]
Thanks, @mamazu! :tada:

**Pull request merged**